mangaki/mangaki/wrappers/anilist.py

```diff
@@ -326,27 +328,27 @@ def _build_external_links(self):
 
     @property
     def staff(self) -> List[Tuple[AniListEntry, str]]:
-        staff_members = []
-        if self.work_info.get('staff'):
-            for staff in self.work_info['staff']:
-                staff_members.append(AniListStaff(
-                    id=staff['id'],
-                    name_first=staff['name_first'],
-                    name_last=staff['name_last'],
-                    role=staff['role']
-                ))
-        return staff_members
+        if not self.work_info.get('staff'):
+            return []
+        return [
+            AniListStaff(
+                id=staff['id'],
+                name_first=staff['name_first'],
+                name_last=staff['name_last'],
+                role=staff['role']
+            ) for staff in self.work_info['staff']
```

Comment:
We can do even better here in terms of conciseness.
Just do `self.work_info.get('staff', [])` and you're done, it'll return an empty list if you iterate over an empty list :).